wp-content/themes/vf-wp-intranet/single-people.php

```diff
@@ -33,7 +33,7 @@
   <div class="vf-content">
     <article class="vf-profile vf-profile--medium vf-profile--inline">
       <img class="vf-profile__image" src="<?php echo esc_url($photo); ?>" alt="" loading="lazy">
-      <h3 class="vf-profile__title">
+      <h3 class="vf-profile__title" style="font-size: 24px;">
```

Comment:
Don't use any hardcoded fonts, make use of utility classes, for above one you can could have used `vf-u-type__text-button--1` which is 24px.

Check more utility classes here - https://stable.visual-framework.dev/assets/vf-utility-classes/vf-utility-classes.css

https://stable.visual-framework.dev/components/vf-utility-classes/index.html